CMakeLists.txt

```diff
@@ -5,39 +5,82 @@ add_definitions(-W -Wall -Wextra)
 
 ## Enforce that we use C++17
 set(CMAKE_CXX_STANDARD 17)
-
-
-find_package(catkin)
-if(${catkin_FOUND})
-  include_directories(${catkin_INCLUDE_DIRS})
-  link_directories(${catkin_LIBRARY_DIRS})
-
-
-  catkin_package(
-    INCLUDE_DIRS include
-    LIBRARIES ${PROJECT_NAME}
-  )
-endif()
+set(CONFIG_NAMESPACE_CMAKE "betwo")
+set(PACKAGE_VERSION 0.0.0)  # Use git tags ??
```

Comment:
We could read the `package.xml` file here, I'll push this in a second

Reply:
I stole the csapex logic for reading the version from `package.xml`.
Do you think this is good enough?

Reply:
Nice! 👍